### ISSUE TITLE
Can't update core-plot release 2.3 with carthage

### ISSUE THREAD

**Issue by result0924** [core-plot/core-plot#411]
I use cathage update core-plot
Checking out core-plot at `47e0af94b4416ed93a6f5b9a58a718e12471502a`

have 1 error 
`framework/Source/CPTLayer.m:344:34: error: no visible @interface for 'UITraitCollection' declares the selector 'performAsCurrentTraitCollection:'
                [traitCollection performAsCurrentTraitCollection: ^{
                 ~~~~~~~~~~~~~~~ ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~`
1 error generated.

---
I can update core-plot release 2.3 with commit `a47d8e1feaffb92585ccdd7c93447b5498105b59`

Any help would be appreciated.

Thanks in Advance.

**Comment by eskroch** [comment]
Which version of Xcode are you using? You may need Xcode 11 with the iOS 13 SDK in order for this to work. You can use an earlier deployment target but the compiler wants to see that selector defined somewhere.

**Comment by eskroch** [comment]
Fixed on the `release-2.3` branch. I added an `@available()` check so this shouldn't require the iOS 13 SDK any more.

**Comment by result0924** [comment]
same error happen, but thanks  for your support
i will test it in XCode 11 with iOS 13 SDK

**Comment by DivineDominion** [comment]
Works for me with Xcode 11 and macOS target platform

**Comment by result0924** [comment]
I use Xcode 10.3 fail, but not serious if Xcode 11 can work

**Comment by koenvanderdrift** [comment]
Seeing this as well; release 2.3, Xcode 10.1, Deployment Target 12.1. My Mac is unfortunately too old to run Xcode 11 and higher.

**Comment by eskroch** [comment]
@koenvanderdrift Do you know a way to fix this for Xcode 10 that will still use the new iOS 13 API when available (i.e., with Xcode 11 and later)

**Comment by koenvanderdrift** [comment]
I don't, but will look into it.

**Comment by koenvanderdrift** [comment]
Finally found a fix for this, see the accepted answer here: https://stackoverflow.com/questions/58083310/if-availableios-13-0-doesnt-compile-in-xcode-10-3. 

On older Xcode versions, and additional check is needed for the OS version:

`#ifdef __IPHONE_13_0
if (@available(iOS 13.0, *)) {
   // do something for iOS 13.0
    else
#else
{
#endif
        // do something for other OS versions
}`

**Comment by eskroch** [comment]
Fixed on the `release-2.4` branch.
